lib/babe/build_integration_test.go

```diff
@@ -100,6 +100,7 @@ func createTestBlock(t *testing.T, babeService *Service, parent *types.Header,
 	return block
 }
 
+// TODO: add test against latest dev runtime
```

Comment:
Maybe epic where we have issues for writing these test on a per-package basis? Like how we did the unit test refactor

Reply:
I created the issues.. I can group them into an epic if needed.